wallet/src/de/schildbach/wallet/ui/main/WalletTransactionsFragment.java

```diff
@@ -433,12 +440,19 @@ private void updateSyncState(BlockchainState blockchainState) {
         } else {
             syncingText.setVisibility(View.VISIBLE);
             String syncing = getString(R.string.syncing);
-            SpannableStringBuilder str = new SpannableStringBuilder(syncing + " " + percentage + "%");
-            int start = syncing.length() + 1;
-            int end = str.length();
-            str.setSpan(new android.text.style.StyleSpan(android.graphics.Typeface.BOLD), start, end,
-                    Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
-            syncingText.setText(str);
+
+            if (percentage == 0) {
+                syncing += "…";
+            } else {
+                SpannableStringBuilder str = new SpannableStringBuilder(syncing + " " + percentage + "%");
```

Comment:
I wonder if this will render correctly on RTL locales.

Reply:
RTL locales show it the same way - "Syncing 5%" with the percentage on the right, but I don't know if it's supposed to be like this. Should RTL language place the percentage on the left?

Seems like the purpose of this `SpannableStringBuilder` is to make the percentage bold. This should be possible to do in the .xml strings with `<b>` tag. Perhaps we can do that and let the translators decide the arrangement.

Reply:
I don't know how the `%` should be displayed on an RTL language.  We can figure this out later.